docs/sources/http_api/folder.md

```diff
@@ -237,7 +237,9 @@ Content-Length: 97
 
 `DELETE /api/folders/:uid`
 
-Deletes an existing folder identified by uid together with all dashboards stored in the folder, if any. This operation cannot be reverted.
+Deletes an existing folder identified by uid together with all dashboards (and their alerts) stored in the folder, if any. This operation cannot be reverted.
+
+If [Grafana 8 Alerts]({{< relref "../alerting/unified-alerting/_index.md" >}}) are enabled, you can set the optional query parameter `forceDeleteRules=false` so that requests will fail with 400 (Bad Request) if the folder contains any Grafana 8 Alerts. If this parameter is set to `true` then it will delete any Grafana 8 Alerts under this folder.
```

Comment:
```suggestion
If [Grafana 8 Alerts]({{< relref "../alerting/unified-alerting/_index.md" >}}) are enabled, you can set an optional query parameter `forceDeleteRules=false` so that requests will fail with 400 (Bad Request) error if the folder contains any Grafana 8 Alerts. However, if this parameter is set to `true` then it will delete any Grafana 8 Alerts under this folder.
```